playground/tree-sitter/build.py

```diff
@@ -175,8 +224,32 @@ def main():
     print("Instantiating index.html.template to dist directory...")
     dist_dir.mkdir(parents=True, exist_ok=True)
     instantiate_index_html(_HERE / "index.html.template", dist_dir / "index.html")
+    
+    # Copy concrete syntax integration files
+    print("Copying concrete syntax integration files...")
+    concrete_syntax_files = ["concrete-syntax.css", "concrete-syntax.js"]
```

Comment:
Sorry, can we rename this file to `concrete-syntax-playground.js` or something similar? 

We have `concrete_syntax.js` which is a wrapper around the WASM, here we have another `concrete-syntax.js` which is gluing the logic to playground :)